docs/reference/functions.txt

```diff
@@ -16,3 +16,7 @@ Functions
    /reference/function/add_logger
    /reference/function/remove_logger
    /reference/function/with_transaction
+
+- :phpmethod:`MongoDB\\add_logger()`
+- :phpmethod:`MongoDB\\remove_logger()`
+- :phpmethod:`MongoDB\\with_transaction()`
```

Comment:
What does `toctree` actually do if it's necessary to duplicate the list of pages in an RST here? Previously, `toctree` was visible in the rendered page output. After the Snooty migration, does it now _only_ control the rendering of the left-hand navigation?

Reply:
Yes I believe it just affects the organization of the left-hand navigation